### DISCUSSION TITLE
Is Modded Sponge also available?

### DISCUSSION THREAD

**Discussion by AquaCobell** [itzg/docker-minecraft-server#1759] [Q&A] [answered]
In the documentation theres only Vanilla Sponge.

**Comment by itzg** [comment, marked as answer]
Do you mean [SpongeForge](https://docs.spongepowered.org/stable/en-GB/server/getting-started/implementations/spongeforge.html)?  If so, you'll use TYPE=FORGE and add SpongeForge as a mod.

**Reply by AquaCobell** [reply]
Youre right, I should have read the docs, thanks
